python/fapolicy_analyzer/ui/ancillary_trust_file_list.py

```diff
@@ -46,17 +47,20 @@ def reducer(map, path):
         )
 
     def _columns(self):
-        statusColumn = Gtk.TreeViewColumn(
+        self.changesColumn = Gtk.TreeViewColumn(
             strings.FILE_LIST_CHANGES_HEADER,
             Gtk.CellRendererText(background="light gray"),
             text=4,
         )
-        statusColumn.set_sort_column_id(4)
-        return [statusColumn, *super()._columns()]
+        self.changesColumn.set_sort_column_id(4)
+        return [self.changesColumn, *super()._columns()]
 
     def load_trust(self, trust):
         changesetMap = self._changesets_to_map()
 
+        # Hide changes column if there are no changes
```

Comment:
I think from the occasional user's perspective or when support is being provided virtually, it would make sense to always have it visible for the sake of consistency and expectations. Consider the case of a user searching for a particular display, "the one with the changes column...," then again I just might be that confused user.